README.md

```diff
@@ -140,6 +140,7 @@ emojis:
 - [hamsterdance](https://raw.githubusercontent.com/snipe/hamsterdance-emojipack/master/hamsterdance.yaml) ([snipe/emojipacks](https://github.com/snipe/hamsterdance-emojipack))
 - [avengers](https://raw.githubusercontent.com/lambtron/emojipacks/master/packs/avengers.yaml)
 - [Shiba Stickers](https://raw.githubusercontent.com/lambtron/emojipacks/master/packs/shiba.yaml) (from Messenger)
+- [gamedevmoji](https://github.com/niksudan/gamedevmoji/blob/master/gamedevicons.yaml)
```

Comment:
Please could you link to the raw file instead, like the others?

https://raw.githubusercontent.com/niksudan/gamedevmoji/master/gamedevicons.yaml